pkg/beyla/config.go

```diff
@@ -222,12 +227,17 @@ func (c *Config) Validate() error {
 			" purposes, you can also set BEYLA_NETWORK_PRINT_FLOWS=true")
 	}
 
+	if c.Printer.Enabled() && c.JSONPrinter.Enabled() {
```

Comment:
We need a config for this new code. The current config tests are in config_test.go.

Reply:
Done